frontend/components/transaction/TransactionFeesTable.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div v-if="isWalletConnected" class="flex flex-col space-y-1 text-gray-700 dark:text-gray-100">
+        <div v-if="swapTransactionFeeETH" class="flex justify-between">
+            <div>Swap Transaction Fee</div>
+            <div>
+                <FormatCurrency :value="swapTransactionFeeETH" :decimals="5" :currency="ETH" />
+            </div>
+        </div>
+        <div v-else-if="bidTransactionFeeETH" class="flex justify-between">
+            <div>Bid Transaction Fee</div>
+            <div>
+                <FormatCurrency :value="bidTransactionFeeETH" :decimals="5" :currency="ETH" />
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Wallet Authorization Fee <Icon v-if="isWalletAuthed" type="check" class="text-green-500" /></div>
+            <div>
+                <FormatCurrency :value="authTransactionFeeETH" :decimals="5" :currency="ETH" />
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>
+                Collateral Authorization Fee <Icon v-if="isCollateralAuthed" type="check" class="text-green-500" />
+            </div>
+            <div>
+                <FormatCurrency :value="authTransactionFeeETH" :decimals="5" :currency="ETH" />
+            </div>
+        </div>
+        <hr />
+        <div class="flex justify-between font-bold">
+            <div>Total</div>
+            <div>
+                <FormatCurrency
+                    v-if="combinedSwapFeesETH"
+                    :value="combinedSwapFeesETH"
+                    :decimals="5"
+                    :currency="ETH"
+                />
+                <FormatCurrency
+                    v-else-if="combinedBidFeesETH"
+                    :value="combinedBidFeesETH"
+                    :decimals="5"
+                    :currency="ETH"
+                />
+            </div>
+        </div>
+    </div>
+    <div v-else>Connect a wallet for an approximate fee breakdown</div>
```

Comment:
I think we can already display some approximate values here. We could break down the different fees, but remove the combined transaction fee at the bottom. 

This way the user can get more insight into how transaction fees look at the current moment.

Reply:
The auth fees are always the same. The only variable is will they apply. If the user has their wallet authenticated we just don't add them to the list. But if say both of us did not do the authorisation and we use the UI, we will have the same values. 

You can find all Gas Prices on our dashboard here:
https://unified-auctions.makerdao.com/dashboard

So users can use that link to see the prices in advance (this also works without a wallet connected). You can see your component as a "micro" version of the dashboard integrated into the flow.